src/Components/QuickSelect.tsx

```diff
@@ -1,202 +1,218 @@
+            </Box>
+          </Box>
+        </Box>
+        <Box ml={2} mt={2} className={globals.classes.flexColumn}>
+          <Divider light />
+          <Box mt={1} />
+          <Typography color="textPrimary" variant="subtitle1">
+            Commonly used
+          </Typography>
+          <Box className={globals.classes.flexRow}>
+            <Box className={globals.classes.flexColumn}>
+              {commonlyUsedColumns[0].map((object, index) => (
+                <Box key={commonlyUsedColumns[0][index]}>
+                  <Button
+                    onClick={() => handleClick(object)}
+                    color="primary"
+                    variant="text"
+                    size="small"
+                    disableFocusRipple={true}
+                    disableRipple={true}
+                    className={globals.classes.quickSelectContainerButton}
+                  >
+                    {""}
+                  </Button>
+                  <Box mt={-3}>
+                    <Typography color="primary" variant="subtitle2">
+                      {MSFormatter.millisecondsToHumanized(
+                        MSFormatter.splitMilliseconds(object),
+                        TermContext.quickSelect,
+                        globals.localeObj
+                      ).join(" ")}
+                    </Typography>
+                  </Box>
+                </Box>
+              ))}
+            </Box>
+            <Box className={globals.classes.flexColumn}>
+              {commonlyUsedColumns[1].map((object) => (
+                <Box key={commonlyUsedColumns[1].indexOf(object)}>
+                  <Button
+                    onClick={() => handleClick(object)}
+                    color="primary"
+                    variant="text"
+                    size="small"
+                    disableFocusRipple={true}
+                    disableRipple={true}
+                    className={globals.classes.quickSelectContainerButton}
+                  >
+                    {""}
+                  </Button>
+                  <Box mt={-3}>
+                    <Typography color="primary" variant="subtitle2">
+                      {MSFormatter.millisecondsToHumanized(
+                        MSFormatter.splitMilliseconds(object),
+                        TermContext.quickSelect,
+                        globals.localeObj
+                      ).join(" ")}
+                    </Typography>
+                  </Box>
+                </Box>
+              ))}
+            </Box>
+          </Box>
+        </Box>
+        <Box ml={2} mt={2} className={globals.classes.flexColumn}>
+          <Divider light />
+          <Box mt={1} />
+          <Typography color="textPrimary" variant="subtitle1">
+            Recently used date ranges
+          </Typography>
+          <Box mt={1} className={globals.classes.flexRow}>
+            <Box className={globals.classes.flexColumn}>
+              {recentlySelectedColumns[0].map((object) => (
```

Comment:
This would let you use the same object as mentioned above to avoid additional duplication.  You could even create a wrapping component that takes a Array[2] and renders the two columns, which would simplify this code even further.

Reply:
Done